issuer/tests/issue_credentials.rs

```diff
@@ -658,7 +671,15 @@ fn should_issue_vc_and_validate_e2e() -> Result<(), CallError> {
             env.time().duration_since(UNIX_EPOCH).unwrap().as_nanos(),
         )
         .expect("credential verification failed");
-        let vc_claims = claims.vc().expect("missing VC claims");
+        let vc_claims = claims
```

Comment:
nit, maybe for a future extension to `ic-verifiable-credentials`: maybe the crate should contain a helper to extract the claims?